cmd/pebble/replay.go

```diff
@@ -60,6 +60,8 @@ func initReplayCmd() *cobra.Command {
 		&c.streamLogs, "stream-logs", c.streamLogs, "stream the Pebble logs to stdout during replay")
 	cmd.Flags().BoolVar(
 		&c.ignoreCheckpoint, "ignore-checkpoint", c.ignoreCheckpoint, "ignore the workload's initial checkpoint")
+	cmd.Flags().StringVar(
+		&c.checkpointDir, "checkpoint-dir", c.checkpointDir, "path to the checkpoint to use if not <WORKLOAD_DIR>/checkpoint")
```

Comment:
could you explain, perhaps in the commit message, the workflow you were envisioning here? This patch is to prevent re-cloning the workload from the snapshot right? I was imagining that a user would run 
```
replay ... --count=10 --checkpoint-dir=foo
```
to slowly clone the checkpoint on the first trial, and then reuse the checkpoint on subsequent runs? Maybe I'm confused on the purpose of this patch.